src/dosbox.cpp

```diff
@@ -615,29 +615,36 @@ void DOSBOX_Init(void) {
 #endif
 #if C_FLUIDSYNTH
 		"fluidsynth",
+#endif
+#if C_MT32EMU
+		"mt32",
 #endif
 		"none",
-		0
-	};
+		0 };
+
 	pstring->Set_values(midi_devices);
 	pstring->Set_help(
 	        "Device that will receive the MIDI data (from the emulated MIDI\n"
-	        "interface - MPU-401).\n"
+	        "interface - MPU-401). Choose one of the following:\n"
```

Comment:
I needed to double-check, because it seemed bizarre to me that MT-32 module would be plugged into MPU-401 module on the real hardware but yup - that's exactly how it worked :)

LGR video was helpful as always https://www.youtube.com/watch?v=JiZOxUhQj10 :)